aws/table_aws_eventbridge_rule.go

```diff
@@ -205,9 +205,11 @@ func listAwsEventBridgeRules(ctx context.Context, d *plugin.QueryData, h *plugin
 
```

Comment:
Please update the GetConfig [here](https://github.com/turbot/steampipe-plugin-aws/pull/1590/files#diff-180c08cb9e58397134b803cc26086099aa46ef8cfbdcc02d0bd3f0ec53c90ddbR20) as below -
```
Get: &plugin.GetConfig{
			KeyColumns: plugin.AllColumns([]string{"name", "event_bus_name"}),
			IgnoreConfig: &plugin.IgnoreConfig{
				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "ValidationException"}),
			},
			Hydrate: getAwsEventBridgeRule,
		},
```